config/config.go

```diff
@@ -729,6 +729,15 @@ func (c *Config) Load(confFile string) error {
 	return err
 }
 
+// MinDDLLease returns the minimum valid value of the DDL lease.
+func (c *Config) MinDDLLease() time.Duration {
+	minDDLLease := time.Duration(0)
```

Comment:
The logic of this function can be simplified?
if c.Store == "tikv" {
return time.Second
}
return time.Duration(0)